SwiftCompilerSources/CMakeLists.txt

```diff
@@ -230,18 +230,28 @@ else()
 
   add_subdirectory(Sources)
 
-  # This is a hack to get correct dependencies from imported C++ headers:
-  # step 1: generate a dummy source file, which just includes all headers defined in include/swift/module.modulemap
-  add_custom_command(
-    OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/HeaderDependencies.cpp"
-    COMMAND
-        "sed"
-        -n -e "/header/!d" -e "s/.*header/#include/" -e "w ${CMAKE_CURRENT_BINARY_DIR}/HeaderDependencies.cpp"
-        "${CMAKE_CURRENT_SOURCE_DIR}/../include/swift/module.modulemap"
-    DEPENDS "${CMAKE_CURRENT_SOURCE_DIR}/../include/swift/module.modulemap"
-    COMMENT "Generate HeaderDependencies.cpp"
-)
-  
+  # TODO: generate this dynamically through the modulemap; this cannot use `sed`
+  # as that is not available on all paltforms (e.g. Windows).
+  #
+  # step 1: generate a dummy source file, which just includes all headers
+  # defined in include/swift/module.modulemap
+  file(WRITE "${CMAKE_CURRENT_BINARY_DIR}/HeaderDependencies.cpp"
+       "\
+#include \"Basic/BridgedSwiftObject.h\" \
```

Comment:
either remove the `\` so that the `#include`s end up on different lines or replace them with `\n`. This just puts them all on the same line.